src/IKubernetes.WebSocket.cs

```diff
@@ -52,7 +52,7 @@ public partial interface IKubernetes
         /// <return>
         /// A <see cref="ClientWebSocket"/> which can be used to communicate with the process running in the pod.
         /// </return>
-        Task<WebSocket> WebSocketNamespacedPodExecAsync(string name, string @namespace = "default", string command = "/bin/bash", string container = null, bool stderr = true, bool stdin = true, bool stdout = true, bool tty = true, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
Let's maintain this signature and add one that takes an `IEnumerable` I don't want to break the API contract for users of this library.